src/main/java/eu/darkbot/api/managers/StarAPI.java

```diff
@@ -0,0 +1,82 @@
+package eu.darkbot.api.managers;
+
+import eu.darkbot.api.API;
+import eu.darkbot.api.entities.utils.Area;
+import eu.darkbot.api.entities.utils.Map;
+import eu.darkbot.api.utils.Listener;
+import eu.darkbot.utils.ArrayUtils;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.Collection;
+import java.util.List;
+
+/**
+ * API to add, manage, retrieve maps.
+ */
+public interface StarAPI extends API {
+    List<String> HOME_MAPS = ArrayUtils.asImmutableList("1-1", "2-1", "3-1");
+    List<String> OUTPOST_HOME_MAPS = ArrayUtils.asImmutableList("1-8", "2-8", "3-8");
```

Comment:
Probably useful to add BLACKLIGHT_MAPS too nowadays since the're used, and maybe PIRATE_MAPS too just for competeness